test/Internal/PsrStreamBodyTest.php

```diff
@@ -16,7 +16,6 @@ class PsrStreamBodyTest extends TestCase
      * @param int|null $size
      * @param int      $expectedSize
      *
-     * @return \Generator
```

Comment:
This line looks like a leftover from v1 - the method returns void.  Unrelated to the compatibility changes - can split this into a separate PR if desired

Reply:
It's fine here.